lib/experimental-default-theme.json

```diff
@@ -133,6 +133,9 @@
 			},
 			"gradient": {
 				"custom": true
+			},
+			"fontSize": {
```

Comment:
I think in other places we use `font-size` but it was inconsistent with other properties on the file, so I chose `fontSize` instead. cc @jorgefilipecosta

Reply:
I think the only place we use kebab-case for this is in the `font-size` preset, for historical reasons. I agree we should update that as well and join team camelCase everywhere.